src/plugins/bannersEverywhere/index.css

```
@@ -0,0 +1,11 @@
+/* stylelint-disable property-no-vendor-prefix */
+.vc-banners-everywhere-memberlist {
+    opacity: 0.8;
+    -webkit-mask-image: linear-gradient(to right, transparent 20%, #fff);
```

you should also add a non prefix variant. likely necessary on firefox, and possibly latest chromium which removed the prefix
```suggestion
    -webkit-mask-image: linear-gradient(to right, transparent 20%, #fff);
    mask-image: linear-gradient(to right, transparent 20%, #fff);
```